### DISCUSSION TITLE
Pushing Content with Chakra-UI Drawer Component

### DISCUSSION THREAD

**Discussion by thecodeclimber** [chakra-ui/chakra-ui#4373] [Q&A]
I am learning how to use the Chakra-UI drawer component:  https://chakra-ui.com/docs/overlay/drawer.  It basically does everything I want it to, with one exception -- it covers the content instead of pushing it to the side.  What I want is a right side drawer that when it opens pushing the content to the left.

Is this possible with the Chakra-UI Drawer component?  If so, how can I do it?

Thanks.

**Comment by anubra266** [comment]
The Drawer works like a modal, so It can't do that. You'll have to create your component, and give it a transition to increase width
